packages/block-editor/src/components/block-list/subdirectory-icon.js

```diff
@@ -3,15 +3,18 @@
  */
 import { SVG, Path } from '@wordpress/components';
 
-const Subdirectory = ( { ...extraProps } ) => (
+const Subdirectory = ( { isRTL, ...extraProps } ) => (
 	<SVG
 		xmlns="http://www.w3.org/2000/svg"
 		width={ 14 }
 		height={ 14 }
 		viewBox="0 0 20 20"
 		{ ...extraProps }
 	>
-		<Path d="M19 15l-6 6-1.42-1.42L15.17 16H4V4h2v10h9.17l-3.59-3.58L13 9l6 6z" />
+		<Path
+			d="M19 15l-6 6-1.42-1.42L15.17 16H4V4h2v10h9.17l-3.59-3.58L13 9l6 6z"
+			transform={ `${ isRTL ? 'scale(-1,1) translate(-20,0)' : '' }` }
```

Comment:
I'm wondering if some actions are taken when we pass an empty string to the `transfrom` and if it wouldn't be better to pass `undefined` instead. I'm just thinking aloud :)